data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml

```diff
@@ -193,6 +193,9 @@
             <column name="hsm_connection_id" type="UUID">
                 <constraints nullable="true"/>
             </column>
+            <column name="virtual_node_state" type="string">
```

Comment:
@benmcmahon100 and I discussed and we agreed to move this new column into `vnode_instance` table.